packages/react-router/src/router.ts

```diff
@@ -1140,6 +1140,7 @@ export class Router<
           scripts: route.options.scripts?.(),
           staticData: route.options.staticData || {},
           loadPromise: createControlledPromise(),
+          fullPath: route.fullPath,
```

Comment:
I think it would be good to have `fullPath` on a match to enable `<Link from={match.fullPath} />`